packages/error-handling/src/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from "./errors";
+export * as utils from "./utils";
```

Comment:
let's put all the exports in a single place
```suggestion
export * from "./utils";
```